parts/openshift/unstable/openshiftmasterscript.sh

```diff
@@ -13,7 +13,7 @@ VERSION="$(rpm -q $SERVICE_TYPE --queryformat %{VERSION})"
 if [ -f "/etc/sysconfig/atomic-openshift-node" ]; then
 	ANSIBLE_DEPLOY_TYPE="openshift-enterprise"
 	IMAGE_TYPE=ose
-	IMAGE_PREFIX="registry.access.redhat.com/openshift3"
+	IMAGE_PREFIX="registry.reg-aws.openshift.com:443/openshift3"
```

Comment:
this PR shouldn't be doing anything with ose for now - we'll sort this out when 3.10 branches

Reply:
Ok, I'll look at getting origin working and circle back around.